crates/ruff_server/docs/MIGRATION.md

```diff
@@ -0,0 +1,82 @@
+## Migrating From `ruff-lsp`
+
+`ruff server`'s configuration is significantly different from `ruff-lsp`, and as such you may need to update your existing configuration.
+
+> \[!NOTE\]
+>
+> The VS Code extension settings have documentation that will tell you whether or not the beta server (`ruff server`) supports a given setting.
```

Comment:
"The VS Code extension settings have documentation that will tell you whether `ruff server` supports a given setting"